<launch>
  <arg name="log_level" default="info" description="Log Level to pass to create_action_servers: debug, info, warn" />

  <!-- Populate the planning scene -->
  <!-- TODO: make it respawn, after I have debugged it -->
  <node pkg="ada_planning_scene" exec="ada_planning_scene" name="ada_planning_scene" args="--ros-args --log-level $(var log_level) --log-level rcl:=INFO --log-level rmw_fastrtps_cpp:=INFO">
    <param from="$(find-pkg-share ada_planning_scene)/config/ada_planning_scene.yaml"/>
    <param name="assets_dir" value="$(find-pkg-share ada_feeding)/assets/"/>
    <remap from="~/face_detection" to="/face_detection" />
    <remap from="~/table_detection" to="/table_detection" />
    <remap from="~/monitored_planning_scene" to="/monitored_planning_scene" />
  </node>

</launch>
